dynamodb/src/it/scala/zio/dynamodb/LiveSpec.scala

```diff
@@ -34,12 +36,18 @@ object LiveSpec extends DefaultRunnableSpec {
     )
   )
 
-//  private val liveAws = http4s.default >>> config.default >>> dynamodb.live >>> DynamoDBExecutor.live
+  private val clockLayer: ZLayer[Any, Nothing, Clock with TestClock] =
```

Comment:
```suggestion
  private val clockLayer = ZLayer.identity[Has[Clock.Service]]
```
This will allow you to push the clock dependency all the way up to be then satisfied with `.provideCustomLayer`